c/bpxskt.c

```diff
@@ -936,7 +941,9 @@ int setSocketBlockingMode(Socket *socket, int isNonBlocking,
           reasonCodePtr);
   
   if (returnValue < 0){
-    printf("BPXFCT failed, ret code %d reason 0x%x\n",*returnCode,*reasonCode);
+    if (socketTrace){
+      printf("BPXFCT failed, ret code %d reason 0x%x\n",*returnCode,*reasonCode);
```

Comment:
This just may not be needed at all, because you will know success by getting a return of NULL.